githubaction/0.1.0/README.md

```diff
@@ -0,0 +1,257 @@
+
+1. In a Keptn project, go to `Uniform page > Uniform`, select the `webhook-service`, and click the `Add subscription` button.
+1. For the subscription section, fill in the following:
+    * **Task**: `mytask-interactive`
+    * **Task suffix**: `triggered`
+1. For the webhook configuration section, fill in the following:
+    * **Request method**: `POST`
+    * **URL**: The API endpoint of the repo: `https://api.github.com/repos/[ORG-NAME]/[REPO-NAME]/dispatches` 
+    * Add a custom header with the following values:
+      * **Name** = `Authorization`
+      * **Value** = `token {{.secret.github-secret.GITHUBTOKEN}}`
+    * Custom Payload must contain the following JSON attributes:
+      * The `event_type` entry is a required attribute with a value that matches one of the event types defined in the GitHub actions workflow file. If the `event_type` value does not match any types defined in any GitHub actions, then nothing happens and the webhook request passes silently. 
+      * The `client_payload` attribute is a JSON payload with extra information about the webhook event that your action or workflow may use
+      * Below is an an custom payload with the `event_type` equal to `interactive`:
+
+      ``` 
+      {
+        "event_type": "interactive",
+        "client_payload": {
+          "type": "{{.type}}",
+          "project": "{{.data.project}}",
+          "service": "{{.data.service}}",
+          "stage": "{{.data.stage}}",
+          "data": "{{.data.custom}}",
+          "shkeptncontext": "{{.shkeptncontext}}",
+          "id": "{{.id}}"
+        }
+      }
+      ```
+
+    * The completed form should look as follows:
+
+      <img src="images/interactive-subscription.png" width="50%" height="50%">
+
+    * Refer to the [GitHub documentation](https://docs.github.com/en/rest/reference/repos#create-a-repository-dispatch-event) for more information on the repository dispatch event.
+
+1. Click the `Create subscription` button to save and enable the webhook for your GitHub Action integration.
+
+1. For a `interactive` webhook, one must also adjust the webhook configuration file to NOT auto-respond with a finished event.  In upcoming releases, this will be configurable from the Bridge UI.  To perform this task, refer to the [Keptn documentation](https://keptn.sh/docs/0.10.x/integrations/webhooks/#configure-webhook-to-not-auto-respond-with-a-finished-event).
+
+1. In order for the GitHub workflow to call the Keptn API, create two [GitHub repository action secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) with the Keptn API URL and Keptn API Token.
+
+    * Go to GitHub repos `Settings > Secret` and click add these two secrets:
+      * `KEPTN_BASE_URL` = Use the base URL for your environment such as: `https://[YOUR-TENANT].cloudautomation.dynatrace.com` for the secret value.
```

Comment:
Please change `cloudautomation.dynatrace.com` to a `demo.keptn.com`

Reply:
fixed in https://github.com/keptn-sandbox/artifacthub/pull/34/commits/2e1e852a46742a14fb2e32877d27e6bf5debf8ae